packages/builders/src/messages/poll/Assertions.ts

```diff
@@ -0,0 +1,25 @@
+import { PollLayoutType } from 'discord-api-types/v10';
+import { z } from 'zod';
+
+export const pollQuestionPredicate = z.object({ text: z.string().min(1).max(300) });
+
+export const pollAnswerEmojiPredicate = z.object({
```

Comment:
hmmm, do we have this anywhere else in our existing assertions? if so, and it's not in the common sort of top-level `Assertions.ts`, then we probably should set one up to re-use

Reply:
Good catch, found one from button-building that I can reuse instead. Will revise.